app/models/manageiq/providers/vmware/cloud_manager.rb

```diff
@@ -0,0 +1,91 @@
+class ManageIQ::Providers::Vmware::CloudManager < ManageIQ::Providers::CloudManager
+  def self.ems_type
+    @ems_type ||= "vmware_cloud".freeze
+  end
+
+  def self.description
+    @description ||= "VMware vCloud".freeze
+  end
+
+  def self.default_blacklisted_event_names
+    []
+  end
+
+  def self.hostname_required?
+    true
+  end
+
+  def supports_port?
+    true
+  end
+
+  def supported_auth_types
+    %w(default)
+  end
+
+  def supports_authentication?(authtype)
+    supported_auth_types.include?(authtype.to_s)
+  end
+
+  #
+  # Connections
+  #
+
+  def self.raw_connect(server, port, username, password)
+    require 'fog/vcloud_director'
+
+    params = {
+      :vcloud_director_username      => username,
+      :vcloud_director_password      => password,
+      :vcloud_director_host          => server,
+      :vcloud_director_show_progress => false,
+      :port                          => port,
+      :connection_options            => {
+        :ssl_verify_peer => false # for development
+      }
+    }
+
+    Fog::Compute::VcloudDirector.new(params)
+  end
+
+  def connect(options = {})
+    raise "no credentials defined" if missing_credentials?(options[:auth_type])
+
+    server   = options[:ip] || address
+    port     = options[:port] || self.port
+    username = options[:user] || authentication_userid(options[:auth_type])
+    password = options[:pass] || authentication_password(options[:auth_type])
+
+    self.class.raw_connect(server, port, username, password)
+  end
+
+  def vcd_service
+    @vcd_service ||= connect
+  end
```

Comment:
@Fryguy agreed I don't think this is needed, in https://github.com/ManageIQ/manageiq/pull/9412 it was used in the refresher but that should just call `ems.connect`.

Side note [RHEV](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/redhat/infra_manager.rb#L92) and [GCE](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/google/manager_mixin.rb#L32) have similar methods, GCE's doesn't seem to be used at all.